src/notebookMiddlewareAddon.ts

```diff
@@ -595,7 +592,12 @@ export class NotebookMiddlewareAddon implements Middleware, Disposable {
 
     public handleDiagnostics(uri: Uri, diagnostics: Diagnostic[], next: HandleDiagnosticsSignature): void {
         const incomingUri = this.converter.toIncomingUri(uri);
-        if (incomingUri && incomingUri != uri && this.shouldProvideIntellisense(incomingUri)) {
+        if (
+            incomingUri &&
+            incomingUri != uri &&
+            this.shouldProvideIntellisense(incomingUri) &&
+            !incomingUri.toString().includes('nteractive') // Skip diagnostics on the interactive window. Not particularly useful
```

Comment:
Can you use `InteractiveScheme` as in the other place?
Also you're missing an `i`

```suggestion
            !incomingUri.toString().includes('interactive') // Skip diagnostics on the interactive window. Not particularly useful
```

Reply:
No the 'i' is on purpose because the URI can have a capital I and a lowercase i depending upon if the input box or the cells.

Reply:
I switched to using a function.